app/main.py

```diff
@@ -68,11 +70,24 @@
 # We define a few attributes of the app object
 app.title = 'Pyronear - Monitoring platform'
 app.config.suppress_callback_exceptions = True
-server = app.server   # Gunicorn will be looking for the server attribute of this module
-
-# We create a rough layout, filled with the content of the homepage
-app.layout = html.Div([dcc.Location(id='url', refresh=False),
-                       html.Div(id='page-content', children=Homepage())])
+server = app.server  # Gunicorn will be looking for the server attribute of this module
+
+# We create a rough layout, filled with the content of the homepage/alert page
+app.layout = html.Div(
+    [
+        dcc.Location(id="url", refresh=False),
+        html.Div(id="page-content", style={"height": "100%"}),
+        # Interval component to generate call to API every 10 seconds
+        dcc.Interval(
+            id="interval-component-homepage",
+            interval=10 * 1000
+        ),
+        # Hidden div to keep a record of live alerts data
+        dcc.Store(id="div_live_alerts_data", storage_type="memory"),
+        dcc.Store(id="last_displayed_event_id", storage_type="memory"),
+        dcc.Store(id="images_url_current_alert", storage_type="memory", data={}),
```

Comment:
When the webhook will be ready I guess that all alerts metadata could be stored in one dcc.Store like `div_live_alerts_data` ?

Reply:
Yes ! But I think that we still need to have `last_displayed_event_id` (to keep track of what is the last event displayed on big screen) and `images_url_current_alert`(dict with event id and a list of images' urls). However for that last one, we will need to add a mechanism to erase past events (probably once the alert has be acquitted).

Reply:
I'm not sure for `images_url_current_alert`, pretty sure that `div_live_alerts_data` can do the job but we'll discuss it later when having a better idea of how we'll interact with webhook data.

Reply:
Ok sure, the webhook will change a bit the workflow anyways !